File: aspnet-core/src/doan.ProjectManagement.Application.Contracts/StudentGroupInformationGroups/Dto/StudentGroupInformationDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.StudentGroupInformationGroups.Dto
{
    public class StudentGroupInformationDto : FullAuditedEntityDto<Guid>
    {
        public Guid? StudentId { get; set; }
        public Guid? StudentGroupId { get; set; }
        public string StudentName { get; set; }
        public string Position { get; set; }
        public string Roles { get; set; }
        public string Description { get; set; }
    }
}

File: applications/teacher-web/src/app/shared/services/student/student-group-information/models/student-group-information-dto.ts
import { EntityDto } from '@abp/ng.core';

export class StudentGroupInformationDto extends EntityDto<string>{

  studentId: string;
  studentGroupId: string;
  studentName: string;
  position: string;
  roles: string;
  description: string;

  constructor(initial: Partial<StudentGroupInformationDto> = {}) {
    super(initial);
  }

}
